tst/PortingAssistantExtensionIntegTests/AssessmentTests.cs

```diff
@@ -210,10 +234,10 @@ private async Task<Boolean> TestSolutionAsync(string[] projectInfo)
                 client = new PAIntegTestClient(solutionPath, solutionName);
 
                 await client.InitClientAsync();
-                var currentResults = await client.AssessSolutionAsync();
+                var currentResults = await client.AssessSolutionAsync(targetFramework);
 
                 /* Uncomment this method to save the results in a json file*/
-                //SaveResults(jsonFile, currentResults);
+                SaveResults(jsonFile, currentResults);
```

Comment:
is this for debugging, should it be commented again?

Reply:
Oh yes, good catch thanks